templates/desktop/organisations.html

```diff
@@ -4,210 +4,227 @@
         hi_def=True,
-        loading="auto"
+        loading="lazy"
         ) | safe
       }}
     </div>
+    <div class="col">
+      <h2>Secure enterprise management with Ubuntu Pro Desktop</h2>
+    </div>
   </div>
-</section>
-
-<section class="p-strip">
   <div class="row">
-    <div class="col-7">
-      <h2 class="u-sv2">Secure enterprise management with Ubuntu Pro Desktop</h2>
+    <div class="col-start-large-4 col-9 col-medium-6">
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Secure open source stack</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p class="p-section--shallow">Receive 10 years of security patches for Ubuntu LTS releases with Expanded Security Maintenance (<a href="/security/esm">ESM</a>) and coverage for over 23,000 packages in the Ubuntu Universe repository. Minimise downtime with Kernel <a href="/security/livepatch">Livepatch</a>.</p>
+          <hr />
+          <a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Landscape fleet management</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p class="p-section--shallow">Software updates, configuration management, policy compliance, and permission control for your entire physical and virtual fleet. Landscape is easy to use and smoothly integrates with other tooling through a robust API.</p>
+          <hr />
+          <a href="/landscape">Learn more about Landscape&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Advanced Active Directory policies</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p class="p-section--shallow">Increase your management capabilities with full Group Policy Object support, remote script execution and privilege management.</p>
+          <hr />
+          <a href="/engage/New-Active-Directory-integration-features">Learn more about Active Directory integration&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Compliance, certification and hardening</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p class="p-section--shallow">Ubuntu Pro delivers security and compliance for even the most sensitive workloads with <a href="/security/certifications#fips">FIPS 140-2 certified modules</a>, <a href="/security/certifications#cis">CIS hardening</a> and <a href="/security/certifications#common-criteria">Common Criteria EAL2</a> certification available.</p>
```

Comment:
The anchor tags in these three headings no longer point to anything. I have left a comment in the doc for Oliver but he is away until the 19th

Reply:
Nice, thanks for catching. Tagged Juan as well just in case he knows